### DISCUSSION TITLE
No healthy upstream response?

### DISCUSSION THREAD

**Discussion by tothj74** [nav-gov-hu/eVAT#60] [Kérdések és válaszok - Gépi interfész] [answered]
Mi lehet az oka, hogy "no healthy upstream" a responsebody minden küldött requestre?

**Comment by renced42** [comment]
Kedves @tothj74 

Még mindig fenn áll a probléma?
Ha igen akkor add fel issue-ba ezt pedig lezárom.

Az issue-be adj meg request ID-t, időpontot, hogy pontosan mit és mikor küldtél.

Köszi!

**Reply by tothj74** [reply, marked as answer]
Kedves @renced42 
Köszönöm a választ.
A jelenség csak egy délután/este volt tapasztalható. Utána már sikeres volt a felküldés.
J.
